reducers/potter.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const potter = createSlice({
+  name: "potter",
+  initialState: {
+    fan: false,
+    loading: false,
+    harryPotterApi: [],
+  },
+
+  reducers: {
+    setHarryPotterApi: (store, action) => {
+      store.harryPotterApi = action.payload;
+    },
+
+    setIntro: (store, action) => {
+      store.fan = action.payload;
+    },
+
+    setLoading: (store, action) => {
+      store.loading = action.payload;
+    },
+  },
+});
+
+//Redux Thunk
```

Comment:
I really like that you incorporate skills from previous weeks into this week, like using redux thunk.  Good job!